install/kubernetes/helm/istio/values.yaml

```diff
@@ -105,6 +105,9 @@ global:
   # propagated, not recommended for tests.
   controlPlaneSecurityEnabled: false
 
+  # indicate the domain used in SPIFFE identity URL
+  identityDomain: cluster.local 
+
```

Comment:
Why should we have n different references to cluster.local ? Why can’t we simplify this and have a single cluster dns suffix field and use it’s value in all these helm charts?

Reply:
This PR is related to #7849, where this parameter is introduced to citadel. Citadel can generate certificates with a configured spiffe trust domain. However, the spiffe URI is checked at many different places in the istio coding to verify that certain certificates are valid. Therefore, we need the value  of identityDomain in a global context.

Maybe the name identityDomain is misleading and should rather be trustDomain.

The value identityDomain is used as the trust domain in certificates for spiffe. It can be different to the cluster dns.   It's main purpose is the usage in multi kubernetes cluster environments.